lua/lualine.lua

```diff
@@ -109,9 +109,12 @@ end
 local function exec_autocommands()
   _G.set_lualine_theme = set_lualine_theme
   _G.set_active_statusline = statusline
+  vim.cmd([[augroup LuaLine]])
```

Comment:
any reason why you are using PascalCase instead of snake_case here?
I think it should simply be `lualine`

Reply:
My bad . Theres no reson . I have a habit of using pascal cases for augroups . I'll fix it.

Reply:
fixed